.github/workflows/push-deploy-dryrun.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy to WordPress.org (Dry Run)
+
+on:
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
```

Comment:
I still run v3 in my scripts, so not sure if v4 will have any negative effects. Might be worth investigating.